photon-client/src/components/dashboard/tabs/CropTab.vue

```diff
@@ -0,0 +1,73 @@
+<script setup lang="ts">
+import { useCameraSettingsStore } from "@/stores/settings/CameraSettingsStore";
+import { type AprilTagPipelineSettings, PipelineType } from "@/types/PipelineTypes";
+import PvSlider from "@/components/common/pv-slider.vue";
+import { computed, getCurrentInstance } from "vue";
+import { useStateStore } from "@/stores/StateStore";
+const currentPipelineSettings = computed<AprilTagPipelineSettings>(
+  () => useCameraSettingsStore().currentPipelineSettings as AprilTagPipelineSettings
+);
+const frame_width = computed(() => useCameraSettingsStore().currentVideoFormat.resolution.width);
+const frame_height = computed(() => useCameraSettingsStore().currentVideoFormat.resolution.height);
+const static_x = computed<number>({
+  get: () => currentPipelineSettings.value.static_x || 0,
```

Comment:
Why might static_x ever be undefined? We have apriltag specific code in AprilTagTab.vue, but we don't need these hacks. Refactor to follow that pattern.